docs/Demos.md

```diff
@@ -7,6 +7,196 @@ title: "React-admin demos"
 
 If you want to see what react-admin is capable of, or if you want to learn from apps built by seasoned react-admin developers, check out these demos.
 
+## Overview
+
+Discover our demos that show you the react-admin potential and help you familiarize with it.
+
+<style>
+    .demos-list {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, 1fr);
+        grid-gap: 10px;
+    }
+
+    @media (max-width: 600px) {
+        .demos-list {
+            display: flex;
+            flex-wrap: wrap;
+            gap: 10px;
+        }
+    }
+
+    .card {
+        box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+        transition: 0.3s;
+        background-color: rgba(243, 246, 249, 0.4);
+        border: 1px solid rgba(0,0,0,0.25);
+        border-radius: 30px;
```

Comment:
`border-radius` is a bit too high IMHO. It doesn't fit well with the rest of the UI, and crops out part of the videos in a weird way, especially on mobile.

![image](https://github.com/marmelab/react-admin/assets/14542336/b2639142-4d5c-4e61-9610-e8740006919f)


But this is a taste issue, you may want to gather more feedback from the team before changing anything.